packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.spec.tsx

```diff
@@ -23,4 +23,30 @@ describe("swirl-table-row-group", () => {
       </swirl-table-row-group>
     `);
   });
+
+  it("renders its label and content", async () => {
+    const page = await newSpecPage({
+      components: [SwirlTableRowGroup],
+      html: `<swirl-table-row-group label="Label" tooltip="Test">Content</swirl-table-row-group>`,
+    });
+
+    expect(page.root).toEqualHtml(`
+      <swirl-table-row-group class="table-row-group" label="Label" role="rowgroup" tooltip="Test">
+        <mock:shadow-root>
+          <div class="table-row-group__header-row" role="row">
+            <span aria-rowspan="0" class="table-row-group__label" role="rowheader">
+              Label
+              <span class="table-row-group__tooltip">
+                <swirl-tooltip content="Test" position="right">
```

Comment:
Quesion: Is "top" the better default position? The component will automatically flip sides if there isn't enough space.

Reply:
yes, matches the design, too